### PR TITLE
Update role-capabilities.md with the new View Connections permission

### PR DESCRIPTION
**Pull request by rmamilla-sumo**
Added documentation for new "view connections" capability allowing admin users to allow right users to view connections.

## Purpose of this pull request

This pull request (PR) ...

Issue number: [SUMO-225496](https://sumologic.atlassian.net/browse/SUMO-225496)
## Select the type of change:

- [ ] New Content - New features, sections, pages, tutorials

### CODE REVIEW

**Comment by jpipkin1** [comment]
I added the "do-not-merge" label.

**Review by jpipkin1 — APPROVED** [review]
The PR looks good, @rmamilla-sumo. Approved.

**Comment by jpipkin1** [comment]
@rmamilla-sumo, note that the "verifiation/cla-signed" check will pass once someone else on my team approves PR https://github.com/SumoLogic/sumologic-documentation/pull/3193 to add you to the contributors list (clabot file).

**Review by stacykor — APPROVED** [review]
LTGM

**Comment by jpipkin1** [comment]
@rmamilla-sumo, I now see the "View Connections" role in the Demo environment.
Screenshot 2023-12-07 at 4 38 03 PM

File: docs/manage/users-roles/roles/role-capabilities.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ Following are the capabilities you can assign when you [create roles](create-ma
 | Manage S3 Data Forwarding	| Manage [S3 data forwarding](/docs/manage/data-forwarding/amazon-s3-bucket) from Sumo Logic to an S3 bucket.|
 | Manage Content	| Manage the content for your organization. This provides access to [Admin Mode](/docs/manage/content-sharing/admin-mode) in the Library.|
 | Manage Apps | Install and manage [apps](/docs/integrations). |
-| Manage Connections | Manage the [connections](/docs/manage/connections-integrations) that allow you to send alerts to other tools. |
+| View Connections | View [connections](docs/alerts/webhook-connections/) on the **Connections** page. |
+| Manage Connections | Manage the [connections](docs/alerts/webhook-connections/) that allow you to send alerts to other tools. |
```
Comment (jpipkin1) [inline docs/manage/users-roles/roles/role-capabilities.md:26]:
```suggestion
| Manage Connections | Manage the [connections](docs/alerts/webhook-connections/) that allow you to send alerts to other tools. |
| View Connections | View [connections](docs/alerts/webhook-connections/) on the **Connections** page. |
```

**Review by jxie418 — APPROVED** [review]

**Pull request merged**